sdk/python/feast/infra/offline_stores/bigquery.py

```diff
@@ -224,19 +223,17 @@ def to_df(self):
         df = self.client.query(self.query).to_dataframe(create_bqstorage_client=True)
         return df
 
-    def to_bigquery(self, dry_run=False) -> Optional[str]:
+    def to_sql(self):
+        return self.query
```

Comment:
Note: there's no option for expiration_timestamp in the [QueryJobConfig](https://googleapis.dev/python/bigquery/latest/generated/google.cloud.bigquery.job.QueryJobConfig.html), so we should also give more granular control for the user to modify and run the SQL directly (like proposal 4 in https://github.com/feast-dev/feast/issues/1654#issuecomment-864218588). An example of this is in my fork [here](https://github.com/codyjlin/feast/blob/offline-to-bq/sdk/python/feast/infra/offline_stores/bigquery.py#L247-L256).

Reply:
I think we are missing docstrings here (and for the other methods). Can you please add it to your method? Also, should there be a return type?